packages/publish/uploadToS3.js

```diff
@@ -1,4 +1,4 @@
-require("dotenv/config")
+require("../../src/loadenv")
```

Comment:
I've updated this require to use `loadenv` but having a hard time figuring out how this gets called/triggered. Could someone provide context around this one?

Reply:
Hokusai triggers the main `package.json` script [here](https://github.com/artsy/force/blob/b6c35ae463eed83ba5a0e99086a96f69c65eaed9/hokusai/config.yml#L2), which then calls out to this sub-package [here](https://github.com/artsy/force/blob/b6c35ae463eed83ba5a0e99086a96f69c65eaed9/packages/publish/package.json#L11).

I'm not sure exactly why this script was moved to this two-call form, but I would advice moving the second call directly into force and avoiding the one-off confusion around nesting mono-repo-esque packages in force (which isn't a mono-repo). 

We used to have this `uploadToS3` file in the root `scripts`, which was simply called from the root package.json. Now theres a few levels of unnecessary indirection.

Reply:
I can understand the intent to _not_ require an entire Force environment (with all of its dependencies) just to upload to S3, or for Force dependencies to be bloated by AWS requirements. On the other hand, if this command is being run from an already-built Force image and it already depends on S3 in other ways, it doesn't make much of a difference.

Either way, assuming this script continues to work, that can be a separate PR.

Reply:
This is to keep CI dependencies out of our production images, they use a significant amount of disk. The perfect solution here would be a separate image that volume mounts force to upload our assets if anyone wants to take that one!